provision.yml

```
@@ -0,0 +1,88 @@
+---
+- name: configure environment
+  hosts: localhost
+  connection: local
+  vars_files:
+    - group_vars/couchbase/vars
+    - group_vars/couchbase/secret_vars
+
+  vars: 
+    count: '{{ servers - 1 }}' # udf 
+    
+  tasks: 
+  - name: check minumum node size
+    fail: 
+      msg: "CouchBase Server requires a minumum of 8GB RAM"
+    when: (type == 'g6-nanode-1') or 
+          (type == 'g6-standard-1') or 
+          (type == 'g6-standard-2') or 
+          (type == 'g6-dedicated-2') or 
+          (type == 'g7-premium-2') | bool
+
+  - name: create couchbase cluster
+    linode.cloud.instance: 
+      label: '{{ instance_prefix }}-{{ item | int +1 }}-{{ uuid }}'
+      api_token: '{{ api_token }}'
+      type: '{{ type }}'
+      region: '{{ region }}'
+      image: '{{ image }}'
+      root_pass: '{{ root_pass }}'
+      authorized_keys: '{{ ssh_keys }}'
+      private_ip: true
+      stackscript_id: 1366193
+      ua_prefix: 'marketplace-couchbase-occ'
+      tags: '{{ cluster_uuid }}'
+      state: present
+    with_sequence: count='{{ count }}'
+
+  - name: collect cluster details
+    linode.cloud.instance_info:
+      api_token: '{{ api_token }}'
+      label: '{{ instance_prefix }}-{{ item | int }}-{{ uuid }}'
+    register: cluster_info
+    with_sequence: count='{{ count | int + 1 }}'
```

```suggestion
    with_sequence: count='{{ servers }}'
```
we already have the cluster count in `servers`